### PR TITLE
Use struct for size config instead of pointer

### PR DESCRIPTION
**Pull request by matthyx**
ref #11517

### CODE REVIEW

File: prow/plugins/size/size.go [hunk 1]
```diff
@@ -230,10 +229,18 @@ func isPRChanged(pe github.PullRequestEvent) bool {
 	}
 }
 
-func sizesOrDefault(sizes *plugins.Size) plugins.Size {
-	if sizes == nil {
-		return defaultSizes
+func defaultIfZero(value, defaultValue int) int {
```
Comment (stevekuznetsov) [inline prow/plugins/size/size.go]: So this is where pointers would be useful -- do we want to support someone setting any of these fields to 0 and not getting defaulted? @cjwagner

Comment (cjwagner) [reply prow/plugins/size/size.go]: I can't think of a reason to specify a value of 0 so I'd say we don't care about supporting that in this case.

Comment (matthyx) [reply prow/plugins/size/size.go]: So we keep it like that?

Comment (stevekuznetsov) [reply prow/plugins/size/size.go]: Keeping it the way you have it SGTM

**Pull request merged**